vendor/k8s.io/kubernetes/pkg/volume/configmap/configmap.go

```diff
@@ -60,7 +60,10 @@ func (plugin *configMapPlugin) GetVolumeName(spec *volume.Spec) (string, error)
 		return "", fmt.Errorf("Spec does not reference a ConfigMap volume type")
 	}
 
-	return volumeSource.Name, nil
+	return fmt.Sprintf(
+		"%v/%v",
```

Comment:
they are both guaranteed to be non-empty, yes.  Strictly speaking, we could drop this patch, but I wanted it for completeness.